saleor/graphql/shipping/mutations/shippings.py

```diff
@@ -260,6 +269,44 @@ def perform_mutation(cls, _root, info, **data):
         )
 
 
+class ShippingZipCodeRulesUpdateInclusionType(BaseMutation):
+    zip_code_rules = graphene.List(
+        ShippingMethodZipCodeRule, description="A shipping method zip code ranges.",
+    )
+    shipping_method = graphene.Field(
+        ShippingMethod, description="Related shipping method."
+    )
+
+    class Arguments:
+        shipping_method_id = graphene.ID(
+            required=True, description="ID of a shipping method to change."
+        )
+        inclusion_type = ZipCodeRuleInclusionTypeEnum(
+            required=True,
+            description="Inclusion type for shipping method zip code rules.",
+        )
+
+    class Meta:
+        description = "Set inclusion type for zip code rules of given shipping method."
+        permissions = (ShippingPermissions.MANAGE_SHIPPING,)
+        error_type_class = ShippingError
+        error_type_field = "shipping_errors"
+
+    @classmethod
+    def perform_mutation(cls, root, info, **data):
+        shipping_method = cls.get_node_or_error(
+            info, data["shipping_method_id"], only_type=ShippingMethod
+        )
+        inclusion_type = data["inclusion_type"]
+
+        shipping_method.zip_code_rules.all().update(inclusion_type=inclusion_type)
```

Comment:
Not sure if this is okay. When admin changes radio button in UI, mutation triggers. From now on all of zip codes I meant to be excluded are included. As an admin I have to delete all zips from the list as it is unlikely I would like to now include all that was excluded and vice-versa. Perhaps im overthinking it and it's not a real problem after all, what do you think?

Reply:
We need to discuss the design with @pwgryglak but I think this mutation may not be needed at all. I think on the design we should have two separate sections: "excluded zip codes" and "included zip codes". Then you would only have to specify the type of a rule when adding it.

Reply:
I'll mark this PR as blocked before we have an agreement.

Reply:
Please let me know when you will decide how to handle it. As I mentioned, I've created this mutation based on current Dashboard design. 
(For me you're proposal sounds better than it @maarcingebala.)